lib/smartcar/vehicle.rb

```diff
@@ -246,8 +246,8 @@ def batch(paths)
     # @return [OpenStruct] An object with a "body" attribute that contains the
     #   response body and a "meta" attribute with the relevant items from response headers.
     def request(method, path, body = {}, headers = {})
-      path = "/vehicles/#{id}/#{path}"
-      raw_response, headers = send(method, path, body, headers)
+      path = "/vehicles/#{id}/#{path.downcase}"
```

Comment:
Uhmmm , isnt it method that we are supposed to downcase. I ignored it in the ode PR when the title said path but the actual change was on the method. I think it should just be change in line 250 to pass `method.downcase`

Reply:
Yes that's correct - thank you for catching this. I should probably write a test next time so I don't make a mistake like that. I'll get this fixed